libadcc/pyiface/export_HartreeFockProvider.cc

```diff
@@ -437,6 +452,10 @@ void export_HartreeFockProvider(py::module& m) {
              "Returns the nuclear multipole of the requested order. For `0` returns the "
              "total nuclear charge as an array of size 1, for `1` returns the nuclear "
              "dipole moment as an array of size 3.")
+        .def("transform_gauge_origin_to_xyz",
+             &HartreeFockProvider::transform_gauge_origin_to_xyz,
+             "Transforms the gauge origin given string to a string containig the "
```

Comment:
"Transforms **a string specifying the gauge origin** to a **tuple** containi**n**g the x, y, z, Ca**r**tesian components."